axoupdater/Cargo.toml

```diff
@@ -10,10 +10,6 @@ readme = "../README.md"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
-[lib]
-name = "axoupdater"
-path = "src/lib.rs"
-
```

Comment:
what happened here? just noticed it could be removed?

Reply:
Yeah, that's right. At first I thought I'd need to change it when adding the new mod, but then I realized it was redundant to begin with now that the workspace has been separated into lib crates and bin crates.